src/app/pipes/record-status.pipe.ts

```diff
@@ -0,0 +1,11 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({ name: 'recordStatusDisplay' })
+export class RecordStatusDisplayPipe implements PipeTransform {
+    transform(value: number) {
```

Comment:
1. I believe `value: RecordStatus` is better than `valud: number`, you can check the `src/app/enums/record-status.enum.ts`, BTW, please update the field in `Record`.
2. Use `switch` instead of multiple `if`.
3. Add a fallback solution when we encountered unexpected value, such as return '未知状态' if no match for all cases.